addons/project/static/src/project_sharing/components/depend_on_ids_one2many/depend_on_ids_list_renderer.js

```diff
@@ -0,0 +1,16 @@
+/** @odoo-module */
+
+import { ListRenderer } from "@web/views/list/list_renderer";
+const { onWillStart } = owl;
+
+export class DependOnIdsListRenderer extends ListRenderer {
+    setup() {
+        super.setup();
+        onWillStart(() => {
+            this.nbHiddenRecords =
+                this.props.list.context.depend_on_count - this.props.list.records.length;
+        });
+    }
```

Comment:
True, and it'll basically get called the same number of time as the setup since the renderer is instantiated each time the notebook page is opened anyway